### ISSUE TITLE
Npm build in local seem has some problem

### ISSUE THREAD

**Issue by bakso** [graphql/graphiql#394]
I want to disable the lint feature, so I remove the source code below in QueryEditor.js:
```js
lint: {
   schema: this.props.schema
},
```
and 
```js
this.editor.options.lint.schema = this.props.schema;
```

Then I builded my package : `npm run build`.
After that I found the code of `graphiql.css` in the root path is quit different with the version which I installed from npm(0.9.3).

The different is shown in the online-compare link: https://www.diffchecker.com/aUsmRSFo

And I inited the source path with a `yarn` command.
What is the problem lead to the case?

**Comment by asiandrummer** [comment]
@bakso whoa, I have no clue - @wincent are you familiar with this maybe?

**Comment by wincent** [comment]
Somewhat familiar. I know we disabled source maps in https://github.com/graphql/graphiql/commit/99d926e5e5245c80f67ee5fbdd1235fd3fcb2c16 which accounts for some of the discrepancies. The other thing is it is normal to expect that a build from `master` will often deviate from the last-released version, given to changes made in commits after the release. @bakso: were you comparing the CSS from the released 0.9.3 package and a local Git checkout of tag v0.9.3?

**Comment by asiandrummer** [comment]
Also I forgot to ask at the time, but does this actually lead to any rendering issues?
I'm going to close this for now due to inactivity - thanks for the report though! Feel free to go ahead and reopen the issue if there is any issues.
